build failed

when I run `mvn clean install` I face the error:
` Errors: 
[ERROR]   DataWriterTest.testWriterForEventData » Mockito Problems setting field client annotated with @org.mockito.Mock(name="", stubOnly=false, extraInterfaces={}, answer=RETURNS_DEFAULTS, serializable=false, lenient=false)
[ERROR]   DataWriterTest.testWriterForItemData » Mockito Problems setting field client annotated with @org.mockito.Mock(name="", stubOnly=false, extraInterfaces={}, answer=RETURNS_DEFAULTS, serializable=false, lenient=false)
[ERROR]   DataWriterTest.testWriterForUserData » Mockito Problems setting field client annotated with @org.mockito.Mock(name="", stubOnly=false, extraInterfaces={}, answer=RETURNS_DEFAULTS, serializable=false, lenient=false)
`

--- a/pom.xml
+++ b/pom.xml
@@ -15,7 +15,7 @@
         <kafka.version>3.3.1</kafka.version>
         <aws.sdk.version>1.12.389</aws.sdk.version>
         <junit.jupiter.version>5.7.0</junit.jupiter.version>
-        <mockito.version>3.6.0</mockito.version>
+        <mockito.version>4.0.0</mockito.version>
         <assertj.version>3.17.2</assertj.version>
         <surefire-plugin.version>3.0.0-M8</surefire-plugin.version>
     </properties>

